client/components/operations/Operation.js

```diff
@@ -38,56 +38,63 @@ export default class Operation extends React.Component {
     render() {
         let op = this.props.operation;
 
-        let rowClassName = op.amount > 0 ? "success" : "";
+        let rowClassName = op.amount > 0 ? 'success' : '';
 
         if (this.state.showDetails) {
-            return <OperationDetails
-                     toggleDetails={this.toggleDetails.bind(this)}
-                     operation={op}
-                     rowClassName={rowClassName} />;
+            return (
+                <OperationDetails
+                  toggleDetails={ this.toggleDetails.bind(this) }
+                  operation={ op }
+                  rowClassName={ rowClassName }
+                />
+            );
         }
 
         // Add a link to the attached file, if there is any.
         let link;
         if (op.binary !== null) {
             let opLink = ComputeAttachmentLink(op);
-            link= <label for={op.id} className="input-group-addon box-transparent">
+            link = (
+                <label htmlFor={ op.id } className="input-group-addon box-transparent">
                     <a
                       target="_blank"
-                      href={opLink}
-                      title={$t('client.operations.attached_file')}>
+                      href={ opLink }
+                      title={ $t('client.operations.attached_file') }>
                         <span className="glyphicon glyphicon-file" aria-hidden="true"></span>
                     </a>
-                  </label>;
+                </label>
+            );
         } else if (op.attachments && op.attachments.url !== null) {
-            let maybeAttachment = <span>
-                <a href={op.attachments.url} target="_blank">
-                    <span className="glyphicon glyphicon-link"></span>
-                    {$t('client.' + op.attachments.linkTranslationKey)}
-                </a>
-            </span>;
+            let maybeAttachment = (
```

Comment:
Wow, there is quite a bug here: maybeAttachment has to be unused. I assume we want something like assigning to `link` here.